### PR TITLE
Add runtime breaking change

### PR DESCRIPTION
**Pull request by Nadeeshan96**
$title
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -97,6 +97,26 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     }
     ```
 
+- Fixed a bug that chose the inherent type incorrectly when constructing a structural value in `value:cloneWithType`
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:100]:
```suggestion
- Fixed a bug that chose the inherent type incorrectly when constructing a structural value in the `value:cloneWithType` 
```

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -97,6 +97,26 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     }
     ```
 
+- Fixed a bug that chose the inherent type incorrectly when constructing a structural value in `value:cloneWithType` 
+and `value:fromJsonWithType` functions. Now, if the target type is a union that includes more than one type descriptor, 
+then the inherent type used will be the first (leftmost) such type descriptor such that a value belonging to that type
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:102]:
```suggestion
then, the inherent type used will be the first (leftmost) such type descriptor such that a value belonging to that type 
```

**Review by praneesha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -97,6 +97,26 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     }
     ```
 
+- Fixed a bug that chose the inherent type incorrectly when constructing a structural value in the `value:cloneWithType`
```
Comment (warunalakshitha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:100]: Should we include issue number?

Comment (warunalakshitha) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:100]: Seems do not need as per other chnages.

**Review by warunalakshitha — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md (hunk 3 above)
Comment (Nadeeshan96) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:100]: Will do that.

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:100]: Not added in other points as well.

**Pull request merged**